plone/app/contentlisting/tests/test_integration_doctest.py

```diff
@@ -1,17 +1,21 @@
 # -*- coding: utf-8 -*-
-from plone.app.contentlisting.tests.base import CONTENTLISTING_FUNCTIONAL_TESTING  # noqa
+
+from plone.app.contentlisting.tests.base import CONTENTLISTING_FUNCTIONAL_TESTING  # NOQA: E501
 from plone.testing import layered
 
 import doctest
 import unittest
 
 
 def test_suite():
-    return unittest.TestSuite([
-        layered(doctest.DocFileSuite(
-            'tests/integration.rst',
-            package='plone.app.contentlisting',
-            optionflags=doctest.NORMALIZE_WHITESPACE | doctest.ELLIPSIS),
-            layer=CONTENTLISTING_FUNCTIONAL_TESTING,
-        ),
-    ])
+    return unittest.TestSuite(
+        [
+            layered(
+                doctest.DocFileSuite(
+                    'tests/integration.rst',
+                    package='plone.app.contentlisting',
+                    optionflags=doctest.NORMALIZE_WHITESPACE | doctest.ELLIPSIS,  # NOQA: E501
```

Comment:
That can be done later after being merged, but usually on testing.py these flags are defined, so you only need to import them.

This has 3 main advantages:
- they can be shared
- on testing.py you can do a multiline statement, so there could be 10000 flags without a problem
- the noqa is not needed anymore 😄